torchvision/prototype/transforms/_misc.py

```diff
@@ -79,3 +81,33 @@ def __init__(self, dtype: torch.dtype, *types: Type) -> None:
 
     def extra_repr(self) -> str:
         return ", ".join([f"dtype={self.dtype}", f"types={[type.__name__ for type in self.types]}"])
+
+
+class _RemoveInvalid(Transform):
```

Comment:
We need a better name for this. Help wanted!

Reply:
`_RemoveEmptyBoundingBoxes`